R/chromote-fbref.R

```diff
@@ -41,14 +41,21 @@ WorldfootballRDynamicPage <- R6::R6Class("WorldfootballRDynamicPage", public = l
 #' @importFrom purrr map_chr
 #' @importFrom xml2 xml_children read_html
 #' @noRd
-worldfootballr_html_page <- function(x) {
-  stopifnot(identical(class(x), c("WorldfootballRDynamicPage", "R6")))
-  nodes <- x$find_nodes("table")
-
-  elements <- purrr::map_chr(nodes, function(node_id) {
-    json <- x$call_node_method(node_id)
-    json$result$value
-  })
+worldfootballr_html_player_table <- function(session) {
+  stopifnot(identical(class(session), c("WorldfootballRDynamicPage", "R6")))
+
+  ## find element "above" commented out table
```

Comment:
note that i couldn't figure out how to identify the element with the commented out table directly, so i've opted to do it "indirectly", by identifying the node above it (which always has the same CSS class).